WordPress/gutenberg
# [Build Tooling]: Fix pre commit hook to build block docs

I don't know if it's just me and is specific to some environment params but I can't commit anything. The error is something like that:

```
npm-run-all api-docs:* found some errors. Please fix them and try committing again.
ERROR: Task not found: "/....../block-library/src/post-featured-image/block.json"
```

What happens here is that we have a pre commit hook that runs [npm-run-all api-docs:*](https://github.com/WordPress/gutenberg/blob/trunk/package.json#L313) but `lint-staged` passes full paths to staged changes so in combination with the `npm-run-all`,  we end up trying to match a `task` from each `staged file`.

In this PR I removed the `npm-run-all` command to avoid the above issue and also changed the `TOKEN` for core blocks documentation in order to be notified by the `are-api-docs-unstaged.js` [here](https://github.com/WordPress/gutenberg/blob/trunk/bin/api-docs/are-api-docs-unstaged.js#L17), which is run right afterwards.

## Testing instructions
1. Commit anything and you will be able to 😄 
2. commit a change in `block.json` - let's say add an attribute and observe that it will not let you because of the updated docs
